src/client.py

```diff
@@ -6,6 +6,9 @@
 
 
 def client(message):
+    """Take a client message, find the servers addr, and sends it to the server.
+    then recieves the servers reply.
+    """
```

Comment:
docstrings should have a one-line, terse command-form and provide extra information as additional lines:

```python
"""Send a message to a server and return the reply from that server

Opens a client socket, connects to server, sends a message and receives a reply.
The reply is returned from the function call
"""
```